docs/DEVSETUP.md

```diff
@@ -1,70 +1,181 @@
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
 
-## Learn More
+## Setup Database and Connection <a name="database"></a>
 
-You can learn more in the [Create React App documentation](https://create-react-app.dev/docs/getting-started/).
+Local Environment Requirements: MongoDB and Node.js
+
+1. **Install MongoDB:**
+   - Install MongoDB either locally or use MongoDB Atlas (MongoDB's cloud-based service).
+   - If installing locally, visit the MongoDB website (https://www.mongodb.com/try/download/community) and download the appropriate version for your operating system.
+   - Follow the installation instructions for your OS to set up MongoDB.
 
-To learn React, check out the [React documentation](https://react.dev/).
+2. **Create an Environment File:**
+   - Create a file named `.env` in the root directory of your Node.js application.
+   - Open the `.variable.env` file, copy its contents, and paste them into your newly created `.env` file.
+
+3. **Configure the Environment Variables:**
+   - In the `.env` file, you'll find configuration variables. Make sure to set the following:
+     - `PORT`: This should be the port number where your Node.js application will run (e.g., `3000`).
+     - `MONGODB_URI`: This should be the URL for your MongoDB instance. If you're using MongoDB locally, it might look like `mongodb://localhost:27017/your-database-name`.
+
+4. **Run the Setup Script:**
+   - Inside your project folder, you may have a folder named `setup`.
+   - Run the `setup.js` file located in that folder. This script is responsible for creating a dummy user in your MongoDB database.
+   - After running the setup script, it should display a message like "User created (email: 'test@gmail.com' password: 'test')."
+
+5. **Start the Node.js Server:**
+   - Now that you have MongoDB set up and the dummy user created, you can start your Node.js server.
+   - Run the `index.js` file or the main entry point of your Node.js application to start the server. You can do this using the `node` command.
+     ```
+     node index.js
+     ```
+   - Your Node.js server should now be running, and it will connect to the MongoDB database using the URL and credentials specified in your `.env` file.
+
+With these steps completed, you should have MongoDB set up, environment variables configured, a dummy user created, and your Node.js server running. You can now start developing and testing your application locally.
+
+You can login in the website with this (email: "test@gmail.com" password: "test")
```

Comment:
add username: test to this email: "test@gmail.com" password: "test" so it would be..

"You can login in the website with this ( username: "test" email: "test@gmail.com" password: "test")"

Reply:
Done

Reply:
Thanks. :+1: :100: